scripts/build/termux_step_install_license.sh

```diff
@@ -1,87 +1,85 @@
+# shellcheck shell=bash
 termux_step_install_license() {
-	[ "$TERMUX_PKG_METAPACKAGE" = "true" ] && return
+	[[ "$TERMUX_PKG_METAPACKAGE" == 'true' ]] && return
 
 	mkdir -p "$TERMUX_PREFIX/share/doc/$TERMUX_PKG_NAME"
-	local LICENSE
-	local COUNTER=0
-	if [ ! "${TERMUX_PKG_LICENSE_FILE}" = "" ]; then
-		INSTALLED_LICENSES=()
+	local LICENSE COUNTER=0
+
+	# Was a license file specified?
+	if [[ -n "${TERMUX_PKG_LICENSE_FILE}" ]]; then
 		COUNTER=1
+		local INSTALLED_LICENSES=()
 		while read -r LICENSE; do
-			[ -z "${LICENSE}" ] && continue
-			if [ ! -f "$TERMUX_PKG_SRCDIR/$LICENSE" ]; then
+			# Skip empty lines
+			[[ -z "${LICENSE}" ]] && continue
+
+			# Check that the license file exists in the source files
+			[[ -f "$TERMUX_PKG_SRCDIR/$LICENSE" ]] || {
 				termux_error_exit "$TERMUX_PKG_SRCDIR/$LICENSE does not exist"
-			fi
-			if [[ " ${INSTALLED_LICENSES[@]} " =~ " $(basename $LICENSE) " ]]; then
+			}
+
+			if [[ " ${INSTALLED_LICENSES[*]} " == *" $(basename "$LICENSE") "* ]]; then
 				# We have already installed a license file named $(basename $LICENSE) so add a suffix to it
-				TARGET="$TERMUX_PREFIX/share/doc/${TERMUX_PKG_NAME}/$(basename $LICENSE).$COUNTER"
-				COUNTER=$((COUNTER + 1))
+				TARGET="$TERMUX_PREFIX/share/doc/${TERMUX_PKG_NAME}/$(basename "$LICENSE").$((COUNTER++))"
 			else
-				TARGET="$TERMUX_PREFIX/share/doc/${TERMUX_PKG_NAME}/$(basename $LICENSE)"
-				INSTALLED_LICENSES+=("$(basename $LICENSE)")
+				TARGET="$TERMUX_PREFIX/share/doc/${TERMUX_PKG_NAME}/$(basename "$LICENSE")"
+				INSTALLED_LICENSES+=("$(basename "$LICENSE")")
```

Comment:
Set `$(basename "$LICENSE")` to a variable instead of it being referenced `4` times.